core/src/main/java/com/yahoo/ycsb/generator/CounterGenerator.java

```diff
@@ -32,23 +32,18 @@
 	public CounterGenerator(int countstart)
 	{
 		counter=new AtomicInteger(countstart);
-		setLastInt(counter.get()-1);
```

Comment:
`setLastInt()` no longer exists. When it did, calling it was pointless anyway because it was mutating an otherwise unused (in this context) value in base class. Last value is always one behind the current one here.